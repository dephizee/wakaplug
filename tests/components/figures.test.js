import React from 'react';
import renderer from 'react-test-renderer';
import { ImageStyle, RotatingImage} from "./../../components/figures.style";


test('ImageStyle working', () => {
  const component = renderer.create(
    <ImageStyle/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('RotatingImage working', () => {
    const component = renderer.create(
      <RotatingImage/>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });