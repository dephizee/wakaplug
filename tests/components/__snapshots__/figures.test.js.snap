// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageStyle working 1`] = `
<img
  alt="..."
  className="sc-bdvvtL iBHpXq"
  src="/logo.png"
/>
`;

exports[`RotatingImage working 1`] = `
<img
  alt="..."
  className="sc-bdvvtL sc-gsDKAQ iBHpXq kXQgLb"
  src="/logo.png"
/>
`;
